RecoTauTag/HLTProducers/test/testL2TauTagNN.py

```diff
@@ -0,0 +1,181 @@
+
+import FWCore.ParameterSet.Config as cms
+from FWCore.ParameterSet.VarParsing import VarParsing
+from Configuration.Eras.Era_Run3_cff import Run3
+import os
+import re
+
+options = VarParsing('analysis')
+options.register('sampleType', 'Run3MC', VarParsing.multiplicity.singleton, VarParsing.varType.string,
+                 "Indicates the sample type: Run3MC or Run2Data")
+#options.register('fileList', '', VarParsing.multiplicity.singleton, VarParsing.varType.string,
+#                 "List of root files to process.")
+#options.register('fileNamePrefix', '', VarParsing.multiplicity.singleton, VarParsing.varType.string,
+#                 "Prefix to add to input file names.")
+options.register('lumiFile', '', VarParsing.multiplicity.singleton, VarParsing.varType.string,
+                 "JSON file with lumi mask.")
+options.register('eventList', '', VarParsing.multiplicity.singleton, VarParsing.varType.string,
+                 "List of events to process.")
+options.register('dumpPython', False, VarParsing.multiplicity.singleton, VarParsing.varType.bool,
+                 "Dump full config into stdout.")
+options.register('Summary', False, VarParsing.multiplicity.singleton, VarParsing.varType.bool,
+                 "want summary")
+
+options.parseArguments()
+dataDict = {"Run3MC" : False, "Run2Data" : True}
+isData = dataDict[options.sampleType]
+processName = 'MLProva'
+process = cms.Process(processName, Run3)
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_cff')
+process.load('HLTrigger.Configuration.HLT_User_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+if not isData:
+    process.load('SimGeneral.MixingModule.mixNoPU_cfi')
+
+process.source = cms.Source('PoolSource', fileNames = cms.untracked.vstring())
+
+# Input source
+'''
+from TauMLTools.Production.readFileList import *
+if len(options.fileList) > 0:
+    readFileList(process.source.fileNames, options.fileList, options.fileNamePrefix)
+elif len(options.inputFiles) > 0:
+    addFilesToList(process.source.fileNames, options.inputFiles, options.fileNamePrefix)
+else:
+    #print(options.inputFiles)
+    #print(options.fileList)
+
+    #process.source.fileNames = cms.untracked.vstring('file:/eos/home-v/vdamante/1E71AB01-1FF5-F049-94D4-642325AFF937.root')  # 5000 evts
+'''
```

Comment:
- keep this functionality
- by default use file from /store and not from your eos area